### PR TITLE
Account-users & application- users Assing/remove functionality

### PR DESCRIPTION
**Pull request by kaushikindianic**

### CODE REVIEW

**Comment by h-wang** [comment]
@kaushikindianic fancy title of the PR :)

**Comment by h-wang** [comment]
@kaushikindianic i think you need to rebase or merge the latest commits first

File: app/schema.xml [hunk 1]
```diff
@@ -50,6 +50,13 @@
         <column name="deleted_at" type="integer" />
         <!-- describe app-types here for SSO -->
     </table>
+    
+     <table name="app_user">
+        <column name="app_name" type="varchar(128)" />
+        <column name="user_name" type="varchar(128)" />
+        <!-- store account specific user data here, like roles, join-status, etc -->
+    </table
```
Comment (joostfaassen) [inline app/schema.xml]: This looks like invalid xml :|

File: src/Repository/PdoAppRepository.php [hunk 2]
```diff
@@ -131,4 +131,33 @@ private function rowToApp($row)
...
+    
+        foreach ($rows as $row) {
+            $aUsers[] = $row['user_name'];
+        }
+        return $aUsers;
+    }
+    
+    public function delAppUsers($appname)
+    {
+        $statement = $this->pdo->prepare('Delete From app_user WHERE app_name = :app_name');
```
Comment (h-wang) [inline src/Repository/PdoAppRepository.php]: suggest to update the deleted_at instead of DELETE

**Pull request merged**